server/admin_management/api/app/repositories/access_control_privilege_repository.py

```diff
@@ -49,3 +48,22 @@ def create_access_control_privilege(
         self.db.flush()
         self.db.refresh(db_item)
         return db_item
+
+    def get_user_delegated_admin_grants(self, user_id: int) -> List[FamRole]:
+        """
+        Find out from `app_fam.fam_access_control_privilege` the applications
+            and roles being granted for the user.
+
+        :param user_id: primary id that is associated with the user.
+        :return: List of "roles" granted for the user or None.
```

Comment:
maybe `List of "roles" the user is allowed to grant or None`?

how about `Find out from `app_fam.fam_access_control_privilege` the roles the user is allowed to grant, and which application the role belongs to`?

Reply:
Thanks for clearer sentence example. Adjusted.